diff --git a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_distortion_ps.h b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_distortion_ps.h
index 98c65f11..777548ef 100644
--- a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_distortion_ps.h
+++ b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_distortion_ps.h
@@ -3,6 +3,7 @@ static const char model_distortion_ps_gl2[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -71,6 +72,8 @@ void main()
     Input.Color = _VSPS_Color;
     vec4 _186 = _main(Input);
     gl_FragData[0] = _186;
+    BODY.gl
+
 }
 
 )";
@@ -80,6 +83,7 @@ static const char model_distortion_ps_gl3[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -149,6 +153,8 @@ void main()
     Input.Color = _VSPS_Color;
     vec4 _186 = _main(Input);
     _entryPointOutput = _186;
+    BODY.gl
+
 }
 
 )";
diff --git a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_renderer_distortion_PS.h b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_renderer_distortion_PS.h
index 00f753c6..a3542ccd 100644
--- a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_renderer_distortion_PS.h
+++ b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_renderer_distortion_PS.h
@@ -3,6 +3,7 @@ static const char model_renderer_distortion_PS_gl2[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -185,6 +186,8 @@ void main()
     Input.Others = _VSPS_Others;
     vec4 _467 = _main(Input);
     gl_FragData[0] = _467;
+    BODY.gl
+
 }
 
 )";
@@ -194,6 +197,7 @@ static const char model_renderer_distortion_PS_gl3[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -377,6 +381,8 @@ void main()
     Input.Others = _VSPS_Others;
     vec4 _467 = _main(Input);
     _entryPointOutput = _467;
+    BODY.gl
+
 }
 
 )";
diff --git a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_renderer_lighting_texture_normal_PS.h b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_renderer_lighting_texture_normal_PS.h
index 7c8ab738..5cfb184c 100644
--- a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_renderer_lighting_texture_normal_PS.h
+++ b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_renderer_lighting_texture_normal_PS.h
@@ -3,6 +3,7 @@ static const char model_renderer_lighting_texture_normal_PS_gl2[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -223,6 +224,8 @@ void main()
     Input.Others = _VSPS_Others;
     vec4 _542 = _main(Input);
     gl_FragData[0] = _542;
+    BODY.gl
+
 }
 
 )";
@@ -232,6 +235,7 @@ static const char model_renderer_lighting_texture_normal_PS_gl3[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -453,6 +457,8 @@ void main()
     Input.Others = _VSPS_Others;
     vec4 _542 = _main(Input);
     _entryPointOutput = _542;
+    BODY.gl
+
 }
 
 )";
diff --git a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_renderer_texture_PS.h b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_renderer_texture_PS.h
index 6fe06c6a..7b12294d 100644
--- a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_renderer_texture_PS.h
+++ b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_renderer_texture_PS.h
@@ -3,6 +3,7 @@ static const char model_renderer_texture_PS_gl2[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -217,6 +218,8 @@ void main()
     Input.Others = _VSPS_Others;
     vec4 _480 = _main(Input);
     gl_FragData[0] = _480;
+    BODY.gl
+
 }
 
 )";
@@ -226,6 +229,7 @@ static const char model_renderer_texture_PS_gl3[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -441,6 +445,8 @@ void main()
     Input.Others = _VSPS_Others;
     vec4 _480 = _main(Input);
     _entryPointOutput = _480;
+    BODY.gl
+
 }
 
 )";
diff --git a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_unlit_ps.h b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_unlit_ps.h
index c9e89ef4..a37151ac 100644
--- a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_unlit_ps.h
+++ b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/model_unlit_ps.h
@@ -3,6 +3,7 @@ static const char model_unlit_ps_gl2[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -43,6 +44,7 @@ void main()
     Input.Color = _VSPS_Color;
     vec4 _65 = _main(Input);
     gl_FragData[0] = _65;
+    BODY.gl
 }
 
 )";
@@ -52,6 +54,7 @@ static const char model_unlit_ps_gl3[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -93,6 +96,8 @@ void main()
     Input.Color = _VSPS_Color;
     vec4 _65 = _main(Input);
     _entryPointOutput = _65;
+    BODY.gl
+
 }
 
 )";
diff --git a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/sprite_distortion_ps.h b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/sprite_distortion_ps.h
index 92de408e..cc10c57f 100644
--- a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/sprite_distortion_ps.h
+++ b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/sprite_distortion_ps.h
@@ -3,6 +3,7 @@ static const char sprite_distortion_ps_gl2[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -68,6 +69,8 @@ void main()
     Input.PosR = _VSPS_PosR;
     vec4 _182 = _main(Input);
     gl_FragData[0] = _182;
+    BODY.gl
+
 }
 
 )";
@@ -77,6 +80,7 @@ static const char sprite_distortion_ps_gl3[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -143,6 +147,8 @@ void main()
     Input.PosR = _VSPS_PosR;
     vec4 _182 = _main(Input);
     _entryPointOutput = _182;
+    BODY.gl
+
 }
 
 )";
diff --git a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/sprite_lit_ps.h b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/sprite_lit_ps.h
index 5e4126c1..b49795e2 100644
--- a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/sprite_lit_ps.h
+++ b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/sprite_lit_ps.h
@@ -3,6 +3,7 @@ static const char sprite_lit_ps_gl2[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -74,6 +75,7 @@ void main()
     Input.ScreenUV = _VSPS_ScreenUV;
     vec4 _158 = _main(Input);
     gl_FragData[0] = _158;
+    BODY.gl
 }
 
 )";
@@ -83,6 +85,7 @@ static const char sprite_lit_ps_gl3[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -155,6 +158,8 @@ void main()
     Input.ScreenUV = _VSPS_ScreenUV;
     vec4 _158 = _main(Input);
     _entryPointOutput = _158;
+    BODY.gl
+
 }
 
 )";
diff --git a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/sprite_unlit_ps.h b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/sprite_unlit_ps.h
index 2989860d..50e582bb 100644
--- a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/sprite_unlit_ps.h
+++ b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/sprite_unlit_ps.h
@@ -3,6 +3,7 @@ static const char sprite_unlit_ps_gl2[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -46,6 +47,8 @@ void main()
     Input.UV = _VSPS_UV;
     vec4 _65 = _main(Input);
     gl_FragData[0] = _65;
+    BODY.gl
+
 }
 
 )";
@@ -56,6 +59,8 @@ static const char sprite_unlit_ps_gl3[] = R"(
 #extension GL_ARB_shading_language_420pack : require
 #endif
 
+HEADER.gl
+
 struct PS_Input
 {
     vec4 Pos;
@@ -99,6 +104,7 @@ void main()
     Input.UV = _VSPS_UV;
     vec4 _65 = _main(Input);
     _entryPointOutput = _65;
+    BODY.gl
 }
 
 )";
diff --git a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/standard_renderer_PS.h b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/standard_renderer_PS.h
index 3f45b4af..bbc5defc 100644
--- a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/standard_renderer_PS.h
+++ b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/standard_renderer_PS.h
@@ -3,6 +3,7 @@ static const char standard_renderer_PS_gl2[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -178,6 +179,8 @@ void main()
     Input.Others = _VSPS_Others;
     vec4 _413 = _main(Input);
     gl_FragData[0] = _413;
+    BODY.gl
+
 }
 
 )";
@@ -187,6 +190,7 @@ static const char standard_renderer_PS_gl3[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -363,6 +367,8 @@ void main()
     Input.Others = _VSPS_Others;
     vec4 _413 = _main(Input);
     _entryPointOutput = _413;
+    BODY.gl
+
 }
 
 )";
diff --git a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/standard_renderer_distortion_PS.h b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/standard_renderer_distortion_PS.h
index a1163aac..e9d85be0 100644
--- a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/standard_renderer_distortion_PS.h
+++ b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/standard_renderer_distortion_PS.h
@@ -3,6 +3,7 @@ static const char standard_renderer_distortion_PS_gl2[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -185,6 +186,8 @@ void main()
     Input.Others = _VSPS_Others;
     vec4 _467 = _main(Input);
     gl_FragData[0] = _467;
+    BODY.gl
+
 }
 
 )";
@@ -194,6 +197,7 @@ static const char standard_renderer_distortion_PS_gl3[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -377,6 +381,8 @@ void main()
     Input.Others = _VSPS_Others;
     vec4 _467 = _main(Input);
     _entryPointOutput = _467;
+    BODY.gl
+
 }
 
 )";
diff --git a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/standard_renderer_lighting_PS.h b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/standard_renderer_lighting_PS.h
index b89d49af..24d0ab60 100644
--- a/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/standard_renderer_lighting_PS.h
+++ b/Dev/Cpp/EffekseerRendererGL/EffekseerRenderer/ShaderHeader/standard_renderer_lighting_PS.h
@@ -3,6 +3,7 @@ static const char standard_renderer_lighting_PS_gl2[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -188,6 +189,8 @@ void main()
     Input.Others = _VSPS_Others;
     vec4 _466 = _main(Input);
     gl_FragData[0] = _466;
+    BODY.gl
+
 }
 
 )";
@@ -197,6 +200,7 @@ static const char standard_renderer_lighting_PS_gl3[] = R"(
 #ifdef GL_ARB_shading_language_420pack
 #extension GL_ARB_shading_language_420pack : require
 #endif
+HEADER.gl
 
 struct PS_Input
 {
@@ -383,6 +387,8 @@ void main()
     Input.Others = _VSPS_Others;
     vec4 _466 = _main(Input);
     _entryPointOutput = _466;
+    BODY.gl
+
 }
 
 )";
